DHProcess/src/main/java/io/deephaven/process/SystemCpuOshi.java

```diff
@@ -58,8 +58,8 @@ public abstract class SystemCpuOshi implements PropertySet {
     /**
      * Gets the Processor ID. This is a hexidecimal string representing an 8-byte value, normally obtained using the
      * CPUID opcode with the EAX register set to 1. The first four bytes are the resulting contents of the EAX register,
-     * which is the Processor signature, represented in human-readable form by {@link #getIdentifier()} . The remaining
-     * four bytes are the contents of the EDX register, containing feature flags.
+     * which is the Processor signature, represented in human-readable form by getIdentifier(). The remaining four bytes
```

Comment:
@devinrsmith can you offer another suggestion here? You're the original author of this code, and I can't find anywhere in the history of this class (even pre-DHC) where this method existed.

Reply:
Looks like bad copy-paste from `oshi.hardware.CentralProcessor.ProcessorIdentifier#getProcessorID`

Reply:
I would remove it - oshi dependency is implementation not api atm.